src/shared/components/help-article/HelpArticle.scss

```diff
@@ -70,6 +70,11 @@ $textColumnWidth: 450px;
       padding: 0 5px;
     }
   }
+
+  pre {
+    background-color: #F1F2F4; // Add a background color to code snippets
```

Comment:
We use special variables for colours to keep them consistent (see [link](https://github.com/Ensembl/ensembl-client/blob/dev/src/styles/_settings.scss#L37)). In this case, it should be `background-color: $light-grey;`

Reply:
Done